src/vue/logo.js

```diff
@@ -0,0 +1,25 @@
+import Vue from "../../node_modules/vue/dist/vue.esm.browser.js";
+
+Vue.component("logo", {
+    props: [
+        "versionConfigurator",
```

Comment:
Is better `configuratorVersion` to be aligned with `firmwareVersion`.

Reply:
Sure, that's pretty much was my brain fart here 💨